sakai-evaluation-tool/src/webapp/component-templates/render_text_item.html

```diff
@@ -35,8 +35,10 @@ <h5 class="title">
 				<span rsf:id="itemText" class="label">Item text label.</span>
 
 				<!-- Previous
```

Comment:
Looks like the markup for this changed, and this commit is now amending commented-out markup which is obviously not helpful. We'll have to look at what the new markup should be, and it would also be a good idea to remove commented-out markup in the templates.